lib/token.cpp

```diff
@@ -160,6 +164,20 @@ void Token::update_property_isStandardType()
     }
 }
 
+void Token::update_property_char_string_literal()
+{
+    if (!(mTokType == Token::eString || mTokType == Token::eChar)) // Token has already been updated
+        return;
+
+    for (const std::string & p : literal_prefix) {
+        if (((mTokType == Token::eString) && mStr.find(p + "\"") == 0) ||
+            ((mTokType == Token::eChar) && (mStr.find(p + "\'") == 0))) {
```

Comment:
you search throught the entire string for `p + "\""`. That is not very efficient.

I see a few alternatives:
 *  `string::compare`
 *  `string::rfind`
 * create a `startsWith` utility function

.. in the future we will use https://en.cppreference.com/w/cpp/string/basic_string/starts_with

Reply:
Thanks. Apparently I got it right at line 48 but not here.

I chose to use `string::compare` since that's used in 30 or so other places in cppcheck.